CMakeLists.txt

```diff
@@ -781,26 +781,31 @@ IF(CPUINFO_SUPPORTED_PLATFORM AND CPUINFO_BUILD_TOOLS)
   CPUINFO_TARGET_ENABLE_C99(isa-info)
   CPUINFO_TARGET_RUNTIME_LIBRARY(isa-info)
   TARGET_LINK_LIBRARIES(isa-info PRIVATE cpuinfo)
+  INSTALL(TARGETS isa-info RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
 
   ADD_EXECUTABLE(cpu-info tools/cpu-info.c)
   CPUINFO_TARGET_ENABLE_C99(cpu-info)
   CPUINFO_TARGET_RUNTIME_LIBRARY(cpu-info)
   TARGET_LINK_LIBRARIES(cpu-info PRIVATE cpuinfo)
+  INSTALL(TARGETS cpu-info RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
 
   ADD_EXECUTABLE(cache-info tools/cache-info.c)
   CPUINFO_TARGET_ENABLE_C99(cache-info)
   CPUINFO_TARGET_RUNTIME_LIBRARY(cache-info)
   TARGET_LINK_LIBRARIES(cache-info PRIVATE cpuinfo)
+  INSTALL(TARGETS cache-info RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
 
   IF(CMAKE_SYSTEM_NAME MATCHES "^(Android|Linux)$" AND CMAKE_SYSTEM_PROCESSOR MATCHES "^(armv[5-8].*|aarch64)$")
     ADD_EXECUTABLE(auxv-dump tools/auxv-dump.c)
     CPUINFO_TARGET_ENABLE_C99(auxv-dump)
     CPUINFO_TARGET_RUNTIME_LIBRARY(auxv-dump)
     TARGET_LINK_LIBRARIES(auxv-dump PRIVATE ${CMAKE_DL_LIBS} cpuinfo)
+    INSTALL(TARGETS auxv-dump RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
I don't think its a good idea to install this binary, as it is just a development tool

Reply:
@cdluminate

Could you comment on this?